changelog/6474.improvement.md

```diff
@@ -0,0 +1,4 @@
+Adapt the training data reader and emulator for wit.ai to their latest format.
```

Comment:
I think it would be very helpful to link to the wit.ai page where this format is documented

Reply:
Actually, there is no place where the format is described (or I was unable to find it)